src/Algebra/Graph.hs

```diff
@@ -356,10 +344,9 @@ concatg combine = fromMaybe empty . foldr1Safe combine
 -- @
 -- foldg 'empty' 'vertex'        'overlay' 'connect'        == id
 -- foldg 'empty' 'vertex'        'overlay' (flip 'connect') == 'transpose'
--- foldg []    return        (++)    (++)           == 'Data.Foldable.toList'
--- foldg 0     (const 1)     (+)     (+)            == 'Data.Foldable.length'
 -- foldg 1     (const 1)     (+)     (+)            == 'size'
 -- foldg True  (const False) (&&)    (&&)           == 'isEmpty'
+-- foldg False ((==) v)      (||)    (||)           == 'hasVertex v'
```

Comment:
Let's switch to `foldg False (==x)      (||)    (||)           == hasVertex x` to match the testsuite as well as actual `hasVertex` implementation.